chainer/distributions/kl_divergence.py

```diff
@@ -0,0 +1,97 @@
+_KLDIVERGENCE = {}
+
+
+def register_kl(dist1, dist2):
```

Comment:
I feel it's better to define it in `distribution.py` instead of under `distributions`.